vnext/Microsoft.ReactNative/Fabric/Composition/CompositionRootAutomationProvider.h

```diff
@@ -14,6 +15,7 @@ class CompositionRootAutomationProvider : public winrt::implements<
                                               IRawElementProviderFragmentRoot,
                                               IRawElementProviderFragment,
                                               IRawElementProviderSimple,
+                                              IInvokeProvider,
```

Comment:
Is this necessary? The implementation does nothing and I'm not sure people are going to be invoking their root views.

Reply:
That makes sense! Will change!